app/components/shared/site-navigation.tsx

```diff
@@ -3,7 +3,7 @@ import { Navigation } from "./navigation";
 
 export function SiteNavigation() {
   return (
-    <div className="flex justify-center items-center">
+    <div className="flex justify-center items-center flex-none">
```

Comment:
I don't think adding `flex-none` is necessary

Reply:
just removed